src/util/bip32.rs

```diff
@@ -31,12 +31,14 @@ use util::base58;
 use util::key::{PublicKey, PrivateKey};
 
 /// A chain code
+#[derive(Copy, Clone, PartialEq, Eq)]
```

Comment:
I didn't derive PartialOrd, Ord, Hash. 
should I add them? (I don't see an obvious need, except if someone throws these things into a Hashmap)

Reply:
I think they can be added when they will be needed

Reply:
I think we should add them now. It's nice to be able to key hashmaps on everything.

Reply:
Idem. I've bumped into several missing derives lately and it's a bit annoying.

Reply:
Done

Reply:
What about `Hash`?